src/Compilers/CSharp/Portable/CSharpExtensions.cs

```diff
@@ -706,6 +706,58 @@ public static Conversion GetConversion(this IConversionOperation conversionExpre
             }
         }
 
+        /// <summary>
+        /// Gets the underlying <see cref="Conversion"/> information from this <see cref="ICompoundAssignmentOperation"/>. This
+        /// conversion is applied before the operator is applied to the result of this conversion and <see cref="IAssignmentOperation.Value"/>.
+        /// </summary>
+        /// <remarks>
+        /// This compound assignment must have been created from C# code.
+        /// </remarks>
+        public static Conversion GetInConversion(this ICompoundAssignmentOperation compoundAssignment)
```

Comment:
Since both of these methods are public APIs, we should test them directly. I only see a test for the `null` argument case.
You can file a follow-up test issue given deadline.

Reply:
Filed [https://github.com/dotnet/roslyn/issues/23093](https://github.com/dotnet/roslyn/issues/23093).

---
In reply to: [149866255](https://github.com/dotnet/roslyn/pull/23041#discussion_r149866255) [](ancestors = 149866255)